geode-docs/tools_modules/geode_api_compatible_with_redis.html.md.erb

```diff
@@ -17,55 +17,54 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-The Redis API for <%=vars.product_name%> allows <%=vars.product_name%> to function as a drop-in replacement for a
+The <%=vars.product_name%> APIs compatible with Redis allows <%=vars.product_name%> to function as a drop-in replacement for a
 highly-available Redis data store, letting Redis applications take advantage of
 <%=vars.product_name%>’s scaling capabilities without changing their client code. Redis clients connect to a <%=vars.product_name%>
 server in the same way they connect to a Redis server, using a hostname and a port number, with
 optional password authentication.
 
-<img src="../images/redis_api_for_geode.png" class="image" />
+<img src="../images/geode_api_compatible_with_redis.png" class="image" />
 
-## <a id="using-the-api" class="no-quick-link"></a>Using the Redis API for <%=vars.product_name%>
+## <a id="using-the-api" class="no-quick-link"></a>Using the <%=vars.product_name%> API compatible with Redis
 
-The <%=vars.product_name%> cluster must have at least one server that is set up to handle the incoming Redis
-commands.
+The <%=vars.product_name%> cluster must have at least one server that is set up to handle the incoming Redis commands.
 
 Use gfsh to start at least one server with a command of the form:
 
 ```pre
 start server \
   --name=<serverName> \
   --locators=<locatorPort> \
-  --redis-port=<redisPort> \
-  --redis-bind-address=<redisBindAddress> \
-  --redis-password=<redisPassword>
+  --compatible-with-redis-port=<compatibleWithRedisPort> \
+  --compatible-with-redis-bind-address=<compatibleWithRedisBindAddress> \
+  --compatible-with-redis-password=<compatibleWithRedisPassword>
 ```
 
-If any of the options `redis-bind-address`, `redis-password`, or `redis-port` are included, a compatible-with-Redis Geode server will be started.
+If any of the options `compatible-with-redis-bind-address`, `compatible-with-redis-password`, or `compatible-with-redis-port` are included, a Geode server compatible with Redis will be started.
```

Comment:
Geode server compatible with Redis will be started. - Does 'Geode' in this sentence need to be  <%=vars.product_name%> ?